src/qt/intro.cpp

```diff
@@ -206,19 +265,18 @@ bool Intro::showIfNeeded(bool& did_show_intro, bool& prune)
         }
 
         /* If current default data directory does not exist, let the user choose one */
-        Intro intro(0, Params().AssumedBlockchainSize(), Params().AssumedChainStateSize());
-        intro.setDataDirectory(dataDir);
-        intro.setWindowIcon(QIcon(":icons/bitcoin"));
-        did_show_intro = true;
+        intro.reset(new Intro(0, Params().AssumedBlockchainSize(), Params().AssumedChainStateSize()));
+        intro->setDataDirectory(dataDir);
+        intro->setWindowIcon(QIcon(":icons/bitcoin"));
```

Comment:
nit: `QStringLiteral(":icons/bitcoin")` ?

Reply:
Not new code here, but ok.